wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/E2eIdentity/DeveloperDeviceDetailsSettingsSelectionViewModel.swift

```diff
@@ -101,4 +102,33 @@ class DeveloperDeviceDetailsSettingsSelectionViewModel: ObservableObject {
             Self.selectedE2eIdentiyStatus = item.value
         }
     }
+
+    static func e2eIdentityProvider() -> E2eIdentityProviding {
+       if DeveloperDeviceDetailsSettingsSelectionViewModel.isE2eIdentityViewEnabled {
```

Comment:
what about starting by the return statment with a guard let in order to improve readability
```suggestion
       guard DeveloperDeviceDetailsSettingsSelectionViewModel.isE2eIdentityViewEnabled  else {
```

Reply:
done